src/bahmni-helm/templates/apps/backup-services/cron-jobs.yml

```diff
@@ -0,0 +1,254 @@
+{{ if .Values.apps }}{{ if .Values.apps.backup_services }} {{ if .Values.apps.backup_services.enabled }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-mysql-backup
+spec:
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 2
+  schedule: {{ .Values.apps.backup_services.schedule | quote }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: {{ .Release.Name }}-mysql-backup
+            image: {{ .Values.apps.backup_services.mysql.image }}
+            env:
+            - name: DB_HOST
+              valueFrom:
+                configMapKeyRef:
+                    name: {{ .Release.Name }}-openmrs-configs
+                    key: OPENMRS_DB_HOST
+            - name: DB_NAME
+              valueFrom:
+                configMapKeyRef:
+                    name: {{ .Release.Name }}-openmrs-configs
+                    key: OPENMRS_DB_NAME
+            - name: DB_USERNAME
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ .Release.Name }}-openmrs-configs
+                  key: OPENMRS_DB_USER
+            - name: DB_PASSWORD
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ .Release.Name }}-openmrs-configs
+                  key: OPENMRS_DB_PASSWORD
+            imagePullPolicy: IfNotPresent
+            volumeMounts:
+            - name: {{ .Release.Name }}-backup-services-storage
+              mountPath: "/opt/backup"
+              subPath: "mysql"
+          restartPolicy: OnFailure
+          {{- with .Values.apps.backup_services.nodeSelector }}
+          nodeSelector: 
+            {{- toYaml . | nindent 12 }}
+          {{end}}
+          volumes:
+          - name: {{ .Release.Name }}-backup-services-storage
+            persistentVolumeClaim:
+              claimName: {{ .Release.Name }}-backup-services-pvc
+
+
+---
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-postres-backup
```

Comment:
`{{ .Release.Name }}-postres-backup` seems like it should be named `{{ .Release.Name }}-odoo-db-backup`